assets/stylesheets/pages/_packages.scss

```
@@ -1,3 +1,9 @@
+.cta {
+  background-color: var(--color-fill-tertiary);
+  border-radius: 0.5rem;
+  padding: 15px;
```

```suggestion
  padding: 1em;
```
So that it scales with the font size.